docs-chef-io/content/server/v14_0/config_rb_server_optional_settings.md

```diff
@@ -106,6 +106,14 @@ This configuration file has the following general settings:
 :   The topology of the Chef Infra Server. Possible values: `manual`,
     `standalone`, and `tier`. Default value: `'standalone'`.
 
+`opscode_erchef["include_version_in_status"]`
```

Comment:
Perhaps for both these we could just indicate that these will be available with 14.1, and it can be merged now.

Reply:
sure, will update the doc